packages/developer-portal/src/components/pages/app-detail-preview/common/app-authentication-detail.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { Dispatch } from 'redux'
+import { useSelector, useDispatch } from 'react-redux'
+import { requestAuthenticationCode } from '@/actions/app-detail'
+import styles from '@/styles/blocks/app-authentication-detail.scss?mod'
+import { Loader, Content, H5 } from '@reapit/elements'
+import { FaCopy } from 'react-icons/fa'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
```

Comment:
could you change to `import CopyToClipboard from 'react-copy-to-clipboard'` instead. After my PR merge it'll be break CI